docs/source/reference/pre_trained_models.rst

```diff
@@ -132,3 +132,49 @@ to opensearch.
 
     - `Config File <https://artifacts.opensearch.org/models/ml-models/huggingface/sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2/1.0.1/onnx/config.json>`_
 
+- `huggingface/sentence-transformers/paraphrase-mpnet-base-v2 <https://huggingface.co/sentence-transformers/paraphrase-mpnet-base-v2>`_
+
+  - TorchScript Format (version: 1.0.0)
+
+    - `Model File <https://artifacts.opensearch.org/models/ml-models/huggingface/sentence-transformers/paraphrase-mpnet-base-v2/1.0.0/torch_script/sentence-transformers_paraphrase-mpnet-base-v2-1.0.0-torch_script.zip>`_
+
+    - `Config File <https://artifacts.opensearch.org/models/ml-models/huggingface/sentence-transformers/paraphrase-mpnet-base-v2/1.0.0/torch_script/config.json>`_
+
+  - Onnx Format (version: 1.0.0)
+
+    - `Model File <https://artifacts.opensearch.org/models/ml-models/huggingface/sentence-transformers/paraphrase-mpnet-base-v2/1.0.0/onnx/sentence-transformers_paraphrase-mpnet-base-v2-1.0.0-onnx.zip>`_
+
+    - `Config File <https://artifacts.opensearch.org/models/ml-models/huggingface/sentence-transformers/paraphrase-mpnet-base-v2/1.0.0/onnx/config.json>`_
+
+
+- `huggingface/sentence-transformers/distiluse-base-multilingual-cased-v1 <https://huggingface.co/sentence-transformers/distiluse-base-multilingual-cased-v1>`_
+
+  - TorchScript Format (version: 1.0.1)
+
+    - `Model File <https://artifacts.opensearch.org/models/ml-models/huggingface/sentence-transformers/distiluse-base-multilingual-cased-v1/1.0.1/torch_script/sentence-transformers_distiluse-base-multilingual-cased-v1-1.0.1-torch_script.zip>`_
+
+    - `Config File <https://artifacts.opensearch.org/models/ml-models/huggingface/sentence-transformers/distiluse-base-multilingual-cased-v1/1.0.1/torch_script/config.json>`_
+
+- amazon/neural-sparse/opensearch-neural-sparse-tokenizer-v1
+
+  - TorchScript Format (version: 1.0.0)
+
+    - `Model File <https://artifacts.opensearch.org/models/ml-models/amazon/neural-sparse/opensearch-neural-sparse-tokenizer-v1/1.0.0/torch_script/opensearch-neural-sparse-tokenizer-v1-1.0.0.zip>`_
+
+    - `Config File <https://artifacts.opensearch.org/models/ml-models/amazon/neural-sparse/opensearch-neural-sparse-tokenizer-v1/1.0.0/torch_script/config.json>`_
+
+- amazon/neural-sparse/opensearch-neural-sparse-encoding-doc-v1
+
+  - TorchScript Format (version: 1.0.0)
+
+    - `Model File <https://artifacts.opensearch.org/models/ml-models/amazon/neural-sparse/opensearch-neural-sparse-encoding-doc-v1/1.0.0/torch_script/opensearch-neural-sparse-encoding-doc-v1-1.0.0-torch_script.zip>`_
+
+    - `Config File <https://artifacts.opensearch.org/models/ml-models/amazon/neural-sparse/opensearch-neural-sparse-encoding-doc-v1/1.0.0/torch_script/config.json>`_
```

Comment:
Is this model config file valid?
It looks like this package requires some extra fields in configs which are not included in this config file.
Or is it not possible to manually upload sparse models?
https://github.com/opensearch-project/opensearch-py-ml/blob/main/opensearch_py_ml/ml_commons/model_uploader.py#L187